packages/@remirror/extension-link/src/link-extension.ts

```diff
@@ -32,17 +34,31 @@ import {
 import { TextSelection } from '@remirror/pm/state';
 import { isInvalidSplitReason, isRemovedReason, Suggester } from '@remirror/pm/suggest';
 
+const UPDATE_LINK = 'updateLink';
+
 /**
  * Can be an empty string which sets url's to '//google.com'.
  */
 export type DefaultProtocol = 'http:' | 'https:' | '';
 
+interface EventMeta {
+  selection: TextSelection;
+  range: FromToParameter | undefined;
+  doc: ProsemirrorNode;
+  attrs: LinkAttributes;
+}
+
 export interface LinkOptions {
   /**
    * Called when the user activates the keyboard shortcut.
    */
   onActivateLink?: Handler<(selectedText: string) => void>;
 
+  /**
+   * Called when the user activates the keyboard shortcut.
```

Comment:
Probably should update the doc text.

```suggestion
   * Called after the `commands.updateLink` has been called.
```